installing/installing_aws/ipi/installing-aws-outposts.adoc

```diff
@@ -1,13 +1,89 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="installing-aws-outposts"]
-= Installing a cluster with compute nodes on AWS Outposts
+= Extending an AWS VPC cluster into an AWS Outpost
 include::_attributes/common-attributes.adoc[]
 :context: installing-aws-outposts
+:zone-type: AWS Outposts
 
 toc::[]
 
-In {product-title} version 4.14, you could install a cluster on Amazon Web Services (AWS) with compute nodes running in AWS Outposts as a Technology Preview. As of {product-title} version 4.15, this installation method is no longer supported.
+In {product-title} version 4.14, you could install a cluster on Amazon Web Services (AWS) with compute nodes running in AWS Outposts as a Technology Preview. As of {product-title} version 4.15, this installation method is no longer supported. Instead, you can install a cluster on AWS into an existing VPC, and provision compute nodes on AWS Outposts as a postinstallation configuration task.
```

Comment:
I wonder if creating a third directory for post-install tasks would be better? That way this document is still in the core AWS docs?

![Screenshot from 2024-07-31 16-04-09](https://github.com/user-attachments/assets/cddc4c48-f00f-4ccb-8b1d-e1677e9aaea4)

Reply:
IPI
UPI
Post-installation

Reply:
I think it makes sense for this to stay under IPI, since [this section](https://79754--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/ipi/installing-aws-vpc#installing-aws-vpc) is sort of a prerequisite and already lives in that section.